k8s/helmfile/helmfile.yaml

```diff
@@ -145,7 +145,7 @@ releases:
 
   - name: api
     chart: wbstack/api
-    version: "0.30.0"
+    version: "0.31.0"
```

Comment:
As is the chart update is a no-op unless `queue.horizon.enabled` is set, so it should be safe to update this for production as well already.